### ISSUE TITLE
[NeoForge 1.21] Game refuses to load... dependency mismatch?

### ISSUE THREAD

**Issue by NovaLynxie** [Lightning-64/Tide#5]
Hi there! I am trying to load Tide on Minecraft 1.21 using NeoForge 21.0.167 but keep getting the same problem of not being able to load my game due to what appears to be a mod dependency mismatch between the mod loader (NeoForge) and its respective Minecraft version.

I have been trying to debug the problem repeatedly, downgrading and reinstalling the mod. Still, every single time, I encounter a problem where this mod keeps failing to load at the game start due to requiring NeoForge 21.1.23, which makes no sense as the mod I downloaded claims to be compatible across 1.21 **and** 1.21.1.

Please can this be looked into as updating to 1.21.1 is impossible for me with the mods I have in my modpack.
See the crash report for the full list of mods.

Affected mods:
Tide v1.2.0 for 1.21 (tried all versions between 1.2.0 -> 1.3.x, none of them work)

Crash Log: [crash-2024-09-16_13.43.49-fml.txt](https://github.com/user-attachments/files/17164999/crash-2024-09-16_13.43.49-fml.txt)

**Comment by Lightning-64** [comment]
I’ll look into this and try to get it fixed as soon as possible! Thanks for letting me know and sorry about the inconvenience.

**Comment by Lightning-64** [comment]
Ok, so I looked at the crash log, and... are you sure this has anything to do with the tide mod? I'm specifically looking at this bit here:

```
Mod File: /C:/Users/Macaulay/AppData/Roaming/PrismLauncher/instances/novalynxie.neoforge/.minecraft/mods/cavesdelight-1.0.2.1-1.21.x.jar
	Failure message: fml.modloadingissue.missingdependency
	Failure message arg 1: neoforge
	Failure message arg 2: cavesdelight
	Failure message arg 3: [21.1.0,)
	Failure message arg 4: 21.0.167
	Failure message arg 5: Optional.empty
	Mod Version: 1.0.2.1-1.21.x
	Mod Issue URL: NOT PROVIDED
	Exception message: MISSING EXCEPTION MESSAGE
```

It says the crash is happening because of an problem with Caves Delight. In fact, Tide isn't mentioned anywhere in the log.
If you remove that mod from your modpack, does the issue get fixed?

**Comment by NovaLynxie** [comment]
Ah... dang it. I uploaded the wrong crash report. I must have mixed up the two from when I was trying to debug Caves Delight, which had the same dependency problem. Here's the correct crash report.
Modpack setup (actual): [crash-2024-09-27_14.35.30-fml.txt](https://github.com/user-attachments/files/17173778/crash-2024-09-27_14.35.30-fml.txt)
Just Tide mod (testing): [crash-2024-09-28_09.39.07-fml.txt](https://github.com/user-attachments/files/17173781/crash-2024-09-28_09.39.07-fml.txt)

I do apologise for the mistake. I forgot to clean up the crash-reports folder after checking the previous errors.

**Comment by Lightning-64** [comment]
No problem! Just looking at the crash report now, this should be an easy fix. I'll try put out a fixed version shortly.

**Comment by Lightning-64** [comment]
Fixed as of file Tide 1.3.2-1.21.x (NEOFORGE) (fix)!
